### ISSUE TITLE
Hlášení chyb na Tenderapp ignorováno?

### ISSUE THREAD

**Issue by ghmv** [tomasko126/easylistczechandslovak#275]
Dle https://adblock.sk/ se mají problémy hlásit na https://easylist-czech-and-slovak.tenderapp.com/ Ovšem tam vidím, že všechny příspěvky, které se zřejmě posílají na adresu tomastaro@adblock.sk, se vrací už rok! jako nedoručitelné, všechny jsou také vedeny jako otevřené.

Spravuje ještě někdo filtr? 
Spravuje někdo stránky, aby na nich byly aktuální informace, kam hlásit problémy?

**Comment by addisr** [comment]
Ano, filtr je aktivně spravován. I problémy nahlášené na tenderapp.com se snažím řešit jen tam není (možná je jen ji nevidím) možnost reagovat (narozdíl od issues zde) + některé nahlášené věci nemusí být automaticky problém těchto filtrů, takže je ani není možné opravit.
